modules/ingester/ingester_test.go

```diff
@@ -320,7 +320,7 @@ func defaultIngesterModule(t *testing.T, tmpDir string) *Ingester {
 			Local: &local.Config{
 				Path: tmpDir,
 			},
-			Block: &encoding.BlockConfig{
+			Block: &common.BlockConfig{
```

Comment:
this is a tough one. block config is _very_ opinionated for current block and should probably end up in v2?

noted elsewhere, but details like this are making me think a cutover to 2.0 is necessary. tempo 2.0 could presumably read the old blocks but never create new ones. then we could ditch config like this.

Reply:
Wanted to avoid breaking changes to the yaml, and the module layout makes it a little slow to reason about.      Here's an idea:  we could move these fields to a new `v2/BlockConfig` and then move this back and compose the v2 struct into it. That way the yaml is unchanged, but the fields are still versioned.

Reply:
i'm not super concerned about it's current location, so feel free to leave it as is while we slowly moving things. i think we need to sync on bigger plans which will make things like "what do we do with BlockConfig" more apparent. i.e. once a new block format exists are we going to only create the new type? will the user even be able to create the old type?